spec/02-integration/04-admin_api/15-off_spec.lua

```diff
@@ -536,6 +536,45 @@ describe("Admin API #off", function()
         assert.equals(snis.certificate.id, certificates.id)
       end)
 
+      it("snis in certificates without certificate", function()
```

Comment:
Could we add a test with raw declarative config?

Or try `_format_version` with `3.0`?